src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +93,11 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getSex() == People.Sex.MAN
+                        ? p.getAge() <= maleToAge && p.getAge() >= fromAge
```

Comment:
let's move `p.getAge() >= fromAge` outside the ternary operator
because it is a common filter, no matter if a person is male or female

Reply:
should I move it fully outside or can I just use it on the left side?

Reply:
okay, I have made two filters, but I thought it's against the checklist :)